test/functional/pdo_sqlsrv/pdostatement_fetchmode_emulate_prepare.phpt

```diff
@@ -61,9 +61,7 @@ try {
     echo "Now selecting....\n";
     $tsql = "SELECT * FROM [$tableName]";
     $stmtOptions[PDO::ATTR_CURSOR] = PDO::CURSOR_SCROLL;
-    if (isColEncrypted()) {
-        $stmtOptions[PDO::SQLSRV_ATTR_CURSOR_SCROLL_TYPE] = PDO::SQLSRV_CURSOR_BUFFERED;
-    }
+
     $stmt1 = $conn1->prepare($tsql, $stmtOptions);
     $stmt1->execute();
     // The row order in the resultset when the column is encrypted (which is dependent on the encrytion key used)
```

Comment:
Do we still need specific if...else block when AE is enabled?

Reply:
yes, the row order of the result set between ae and off seems to be different